src/main/java/ru/testtask/service/ChatMessageService.java

```diff
@@ -0,0 +1,69 @@
+package ru.testtask.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.messaging.simp.SimpMessageSendingOperations;
+import org.springframework.stereotype.Service;
+import ru.testtask.model.ChatMessage;
+import ru.testtask.model.ChatRoom;
+import ru.testtask.model.MessageType;
+import ru.testtask.repo.MessageRepo;
+
+import java.util.Date;
+import java.util.Optional;
+
+
+import static java.lang.String.format;
+
+@Service
+public class ChatMessageService {
```

Comment:
Please create interface `ChatMessageService`.
1) Rename this to `ChatMessageServiceImpl`
2) This service should implement `ChatMessageService`
3)  Followind methods should be added to interface
   a) void sendUserMessage(String roomId, String username, String content); - this should create message to db and send message to channel
   b) void sendUserJoinToRoom(String roomId, String username);
   c) void sendUserLeaveFromRoom(String roomId, String username);
   d) Page<ChatMessage> findAllByRoom(String roomId);
   e) Page<ChatMessage> findAllByRoom(String roomId, int page, int pageSize);
4) The service should be used outside only via interface